less/index.less

```diff
@@ -57,3 +57,206 @@ html, body {
     height: 100%;
 }
 
+/******************VARIABLES*****************************/
+
+@lighter-color: cornsilk;
+@darker-color: darkslategrey;
+@border-color: grey;
+@font: 'Poppins', sans-serif;
+
+
+/****************** MIXINS *****************************/
+
+.makeItRight() {
+	display: flex;
+	justify-content: flex-end;
+	padding: 2%;
+}
+
+.makeItLeft(@padding){
+	display: flex;
+	justify-content: flex-start;
+	padding: @padding;
+}
+
+/************************** REGULAR CODE *******************/
+
+h2 {
+	font-size: 1.5rem;
+
+	@media screen and (max-width: 500px) {
+		font-size: 1rem;
+	}
+}
+
+p, a, th, tr, h3 {
+	@media screen and (max-width: 500px) {
+		font-size: .9rem;
+	}
+}
+
+p, h2, h1, h3, a, th, tr {
+	font-family: @font;
+}
+
+.container {
+	margin: 0 2%;
+
+		header {
+			.makeItRight();
+
+			@media screen and (max-width: 500px) {
+				display:flex;
+				justify-content: center;
+			}
+
+			nav {
+				width: 40%;
+				display: flex;
+				justify-content: flex-end;
+				justify-content: space-between;
+
+				@media screen and (max-width: 500px) {
+					width: 80%;
+				}
+			
+				a {
+					text-decoration: none;
+					color: black;
+				}
+
+			}
+		}
+
+		.topContent {
+			.makeItLeft (3.5% 2%);
+			flex-direction: column;
+			background: @lighter-color;
+
+				.boxWithImage {
+					width: 30%;
+
+					img {
+						width: 100%;
+					}
+				}
+
+				.aboveTheLine {
+					border-bottom: 1px solid @border-color;
+					padding-bottom: 3%;
+				
+						h1 {
+							font-size: 10vh;
```

Comment:
This isn't really a problem with my code, per se. But I'm wondering if you would have used a different unit of measurement for the font size? I didn't originally have this, but when I scale it, it doesn't move like I want it too.